examples/electron/package.json

```diff
@@ -52,7 +52,8 @@
     "@theia/userstorage": "1.10.0",
     "@theia/variable-resolver": "1.10.0",
     "@theia/vsx-registry": "1.10.0",
-    "@theia/workspace": "1.10.0"
+    "@theia/workspace": "1.10.0",
+    "@theia/terminal-external": "1.10.0"
```

Comment:
minor: should be in alphabetical order (the release will actually re-arrange the dependencies).